src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -2173,5 +2173,17 @@ public override ForEachStatementInfo GetForEachStatementInfo(CommonForEachStatem
             MemberSemanticModel memberModel = GetMemberModel(node);
             return memberModel == null ? default(ForEachStatementInfo) : memberModel.GetForEachStatementInfo(node);
         }
+
+        public override DeconstructionInfo GetDeconstructionInfo(AssignmentExpressionSyntax node)
+        {
+            MemberSemanticModel memberModel = GetMemberModel(node);
+            return memberModel == null ? default : memberModel.GetDeconstructionInfo(node);
```

Comment:
`memberModel?.GetDeconstructionInfo(..) ?? default`? For some reason the ==  null throws me off.